### ISSUE TITLE
NoSuchFieldError with ElectricExpansion

### ISSUE THREAD

**Issue by fogger** [agaricusb/ModAnalyzer#5]
Else Kaboom!

**allmods**
- `BasicComponents_v1.3.1.359.jar`
- `ElectricExpansion_v2.1.0.27.jar`

Obviously not a fault of ModAnalyzer, but maybe you want a special case?

**Comment by agaricusb** [comment]
Yeah I think adding these dep fixes to mcmodfixes.py is the best I can do for now, keep these issues coming :)

Added in https://github.com/agaricusb/ModAnalyzer/commit/3d27b9f4ced6dd81a3f715e8f0c39f8b0e67dbba

but I'm still unable to load EE with: BasicComponents_v1.3.1.67.jar     ElectricExpansion_v2.1.0.27.jar     ModAnalyzer-1.0-SNAPSHOT.jar        UniversalElectricity_v1.3.1.67

fails with: java.lang.NoSuchFieldError: icons
        at electricexpansion.common.items.ItemUpgrade.<init>(ItemUpgrade.java:25)

only search hits on Google:  http://pastebin.com/QQheU7jB (1 day ago) http://pastebin.com/Ljm29PYu (8 hours ago) - mod issue?

**Comment by fogger** [comment]
I have no idea what calclavia is doing over there, but he's creating a butt-load of confusion.

From what I have ascertained:
`UniversalElectricity` is the source code for `BasicComponents` and `BCDev` is a compiled debug version of `BasicComponents`

The only file of interest to Joe Shmoe is `BasicComponents`, as it has been since 1.42, but for 1.5.1, in his infinite wisdom, he decides to remove the "api" suffix from the source code filename & move it to the front of the download list. which is obviously going to result in every average person thinking they need that...

And if that wasn't weird enough, suddenly the build number jumps from 359 to 56 and continues from there.  but I have had no success with any of the artifacts 56 to 67. yet with build 359, no errors & EE appears to be working fine, so I stuck with 359.

When I attempt to run EE with any of the supposed newer BC artifacts, I don't get NoSuchFieldError's, I get a slew of other errors, TypeNotPresentException etc.

At this point I'm ready to give UE some private time to sort things out.

**TL;DR**
try BasicComponents build 359.

**Forgot to mention:** I get NoSuchFieldError's when attempting to run EE without BC in an MMC instance.

**EDIT2:** I altered the modfixes.py to remove the UniversalElectricity entries, and although it did add BC & EE to the temp server, I still got NoSuchFieldError's, which is strange because I also set up a fresh MMC instance with just BC+EE and it works fine.
